virttest/libvirt_storage.py

```diff
@@ -544,3 +544,17 @@ def clone_volume(self, old_name, new_name):
             logging.info("Volume '%s' does not exist or '%s' has been exist."
                          % (old_name, new_name))
             return False
+
+
+def check_qemu_image_locking_feature_support():
+    """
+    QEMU commit 244a566(qemu-2.10) introduced the image locking feature which brought
+    a new option '-U' to the qemu-img command:info, compare, check, bench, convert,
+    dd, map, snapshot, rebase. This method provides one way to determine whether
+    current qemu-img support or not.
+
+    :return: True if current qemu-img command support
+    """
+    cmd_result = process.run("qemu-img" + ' -h', ignore_status=True,
```

Comment:
It is better to check for the binary and proceed
```python
cmd = "qemu-img"
try:
    binary_path = process.system_output("which %s" % cmd)
except process.CmdError:
    raise ...
cmd_result = process.run("%s %s -h" % (binary_path, cmd), ignore_status=True,
                         shell=True, verbose=False)
return '-U' in cmd_result.stdout
```

Reply:
updated